packages/makeswift/src/utils/is-next-app.ts

```diff
@@ -12,7 +12,9 @@ function hasNextDependency(appDirectory: string): boolean {
 
     return 'next' in dependencies
   } catch (error) {
-    if (error instanceof Error) console.log(error.message)
+    if (error instanceof Error) {
+      /* do nothing */
+    }
   }
```

Comment:
Is the `instanceof` check here necessary if we're not doing anything with it?

```suggestion
  } catch {
    // do nothing
  }
```

or

```suggestion
  } catch {
    return false
  }
```

or perhaps even better would be to check for the not found error of `require` and in that case return `false` and avoid a try/catch here. Or even using `fs.exists` before `require`, etc.